src/utils/check.your.answers.ts

```diff
@@ -51,6 +52,11 @@ export const postDataForReview = async (req: Request, res: Response, next: NextF
 
     const session = req.session as Session;
     const appData: ApplicationData = getApplicationData(session);
+    const noChangeReviewStatement = req.body["no_change_review_statement"];
+
+    if (noChangeReviewStatement === "0") {
```

Comment:
For check your answers page it this the case: noChangeReviewStatement === undefined

Reply:
`  items: [
            {
              value: "1",
              text: "Yes, the information is correct",
              checked: no_change_review_statement == 1,
              attributes: {
                "data-event-id": "no-change-review-statements-radio-option-yes"
              }
            },
            {
              value: "0",
              text: "No, I need to make changes",
              checked: no_change_review_statement == 0,
              attributes: {
                "data-event-id": "no-change-review-statements-radio-option-no"
              }
            }`

Reply:
But this is get util method is included in registration journey? So there it's undefined?

Reply:
which should be falsy so would be ok but needs testing?

Reply:
We also need to cover the scenario where neither of the options have been selected and we return a validation error, not sure if we're doing this?

Reply:
That validation should be covered by the "no change statement" page so I'd expect 0/1 for update but undefined for registration.